vscode/src/azure/networkRequests.ts

```diff
@@ -25,6 +25,7 @@ export async function azureRequest(
 
   try {
     log.debug(`Fetching ${uri} with method ${method}`);
+    log.trace("Request headers & body: ", headers, body);
```

Comment:
I added this to help me troubleshoot the issue, so leaving it in for future use. We should capture network requests in 'trace' logging.

Reply:
I love debug/trace logging - thanks for adding this

Reply:
Any potential privacy issues here?